### PR TITLE
Update KD Notebook for classification

### PR DESCRIPTION
**Pull request by BloodAxe**
This PR update KD Notebook and fixes a few bugs related:
1) A predict() was not possible to use with KDTrainer (dataset preprocessing params were not properly set)
2) Resize implementation for classification was implemented wrong (Not as it works in torchvision)
3) Extracting preprocessing transforms from torchvision transforms were not implemented

### CODE REVIEW

File: src/super_gradients/training/datasets/classification_datasets/cifar.py [hunk 1]
```diff
@@ -76,3 +92,17 @@ def __init__(
             target_transform=target_transform,
             download=download,
         )
+
+    def get_dataset_preprocessing_params(self) -> Dict:
```
Comment (shaydeci) [inline src/super_gradients/training/datasets/classification_datasets/cifar.py:96]: So, I was wondering how to handle the fact that we dont actually use any resizing in validation, but images are all sized (64, 64).
If we just rely on the preprocessing set here, we might fail in practice running on external source images.

**Review by shaydeci — CHANGES_REQUESTED** [review]
Just a single comment inline.

**Review by Louis-Dupont — COMMENTED** [review]
Looks good already, I just have a small test sugestion

File: src/super_gradients/training/datasets/classification_datasets/cifar.py (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/training/datasets/classification_datasets/cifar.py:96]: Maybe the right thing to do, is to check whether there is any resizing, and if not add a resize to 64 processing.

Comment (BloodAxe) [reply src/super_gradients/training/datasets/classification_datasets/cifar.py:96]: As aggreed, added Resize operations to val dataset params instead.
Fixed in 5151cd975529f700b91b4190b44f5231932ce0cf

File: README.md [hunk 2]
```diff
@@ -312,7 +312,9 @@ Knowledge Distillation is a training technique that uses a large model, teacher
 Learn more about SuperGradients knowledge distillation training with our pre-trained BEiT base teacher model and Resnet18 student model on CIFAR10 example notebook on Google Colab for an easy to use tutorial using free GPU hardware
   <table class="tfo-notebook-buttons" align="left">
  <td width="500">   
-   <a target="_blank" href="https://bit.ly/3BLA5oR"><img src="./documentation/assets/SG_img/colab_logo.png" /> Knowledge Distillation Training</a>
+   <a target="_blank" href="https://colab.research.google.com/github/Deci-AI/super-gradients/blob/master/notebooks/how_to_use_knowledge_distillation_for_classification.ipynb">
+    <img src="./documentation/assets/SG_img/colab_logo.png" /> Knowledge Distillation Training
```
[the remark this answers is not in the document]
Comment (BloodAxe) [reply README.md:316]: oops, thanks for checking that. Fixed

File: src/super_gradients/training/datasets/classification_datasets/torchvision_utils.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from typing import List, Any, Dict
+
+from torchvision.datasets.vision import StandardTransform
+from torchvision.transforms import Resize, ToTensor, Normalize, CenterCrop, Compose
+
+from super_gradients.common.object_names import Processings
+
+
+def get_torchvision_transforms_equivalent_processing(transforms: List[Any]) -> List[Dict[str, Any]]:
```
[the remark this answers is not in the document]
Comment (BloodAxe) [reply src/super_gradients/training/datasets/classification_datasets/torchvision_utils.py:9]: Added test 98dcbee81d07a88a5ae9398c74e306ea96f5e617
Figured out we have non-matching implementation for Resize operation which was also fixed

**Review by shaydeci — APPROVED** [review]
LGTM

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**